buildconfig.js

```diff
@@ -61,26 +61,23 @@ if (typeof process.env.HELP_LINK === 'string') {
     indexHtml = indexHtml.replace(reZendesk, `<script id="ze-snippet" type="text/javascript" src="${process.env.HELP_LINK}"></script>`);
   }
 }
-
+var matomoJs = fs.readFileSync('matomo.js', 'utf8');
```

Comment:
What is the "justification" to move the matomo js code elsewhere ?

Reply:
1) Keep index.ejs as concise as possible
2) Keep index.ejs as close as possible as Tidepool's version to facilitate future merges.
3) Simplify the build-config process